wifiphisher/common/macmatcher.py

```diff
@@ -57,6 +57,30 @@ def _get_vendor_information(self):
                     logo = separated_line[2]
                     self._mac_to_vendor[mac_identifier] = (vendor, logo)
 
+    def get_random_mac_addrs(self, num_of_mac_addrs):
+        """
+        Make random MAC address and return as a List with len equal
+        to num_of_mac_addrs
+
+        :param self: A MACMatcher object
+        :param num_of_mac_addrs: Number of the MAC addresses needed
+        :type self: MACMatcher
+        :type num_of_mac_addrs: int
+        :return: A List of random MAC address
+        :rtype list
+        """
+        mac_addrs = []
+        ouis = random.sample(self._mac_to_vendor, num_of_mac_addrs)
```

Comment:
Using our own oui's is too much work and overhead. Randomness is what we want and we can simply use something like this:
```python
"02:00:00:{:02x}:{:02x}:{:02x}".format(random.randint(0,255),random.randint(0,255),random.randint(0,255))
```

Which I have taken from [here](http://stackoverflow.com/a/43546406).